<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second_Dialog</class>
 <widget class="QDialog" name="Second_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1048</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 113, 143);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_38">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_33">
     <item row="0" column="0">
      <widget class="QWidget" name="okien_1" native="true">
       <layout class="QGridLayout" name="gridLayout_37">
        <item row="0" column="0">
         <widget class="QWidget" name="okien_11" native="true">
          <property name="minimumSize">
           <size>
            <width>491</width>
            <height>211</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#okien_11{
    background-color: rgb(170, 170, 255);
	border: 1px solid black;
    border-radius: 30px;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_41">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>475</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">    background-color: rgb(170, 170, 255);</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <spacer name="horizontalSpacer_22">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="nazwa_okna11">
                <property name="styleSheet">
                 <string notr="true">QLabel#nazwa_okna11{
	color: rgb(255, 255, 255);
	background-color: rgb(60, 119, 179);
	border: 1px solid black;
    border-radius: 10px;
}</string>
                </property>
                <property name="text">
                 <string>Wysokość poziomu wody</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_21">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="woda_wykres" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="okien_4" native="true">
       <layout class="QGridLayout" name="gridLayout_34">
        <item row="0" column="0">
         <widget class="QWidget" name="okien_44" native="true">
          <property name="minimumSize">
           <size>
            <width>491</width>
            <height>211</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#okien_44{
    background-color: rgb(170, 170, 255);
	border: 1px solid black;
    border-radius: 30px;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_40">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">    background-color: rgb(170, 170, 255);</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_20">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="nazwa_okna44">
                <property name="styleSheet">
                 <string notr="true">QLabel#nazwa_okna44{
	color: rgb(255, 255, 255);
	background-color: rgb(60, 119, 179);
	border: 1px solid black;
    border-radius: 10px;
}</string>
                </property>
                <property name="text">
                 <string>Temperatura</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_19">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="temperatura_wykres" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="okien_2" native="true">
       <layout class="QGridLayout" name="gridLayout_36">
        <item row="0" column="0">
         <widget class="QWidget" name="okien_22" native="true">
          <property name="minimumSize">
           <size>
            <width>491</width>
            <height>211</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#okien_22{
    background-color: rgb(170, 170, 255);
	border: 1px solid black;
    border-radius: 30px;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_42">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">    background-color: rgb(170, 170, 255);</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <spacer name="horizontalSpacer_24">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="nazwa_okna22">
                <property name="styleSheet">
                 <string notr="true">QLabel#nazwa_okna22{
	color: rgb(255, 255, 255);
	background-color: rgb(60, 119, 179);
	border: 1px solid black;
    border-radius: 10px;
}</string>
                </property>
                <property name="text">
                 <string>Prędkość przepływu</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_23">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="przeplyw_wykres" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="okien_3" native="true">
       <layout class="QGridLayout" name="gridLayout_35">
        <item row="0" column="0">
         <widget class="QWidget" name="okien_33" native="true">
          <property name="minimumSize">
           <size>
            <width>491</width>
            <height>211</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#okien_33{
    background-color: rgb(170, 170, 255);
	border: 1px solid black;
    border-radius: 30px;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_39">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>475</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">    background-color: rgb(170, 170, 255);</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="nazwa_okna33">
                <property name="styleSheet">
                 <string notr="true">QLabel#nazwa_okna33{
	color: rgb(255, 255, 255);
	background-color: rgb(60, 119, 179);
	border: 1px solid black;
    border-radius: 10px;
}</string>
                </property>
                <property name="text">
                 <string>Poziom wilgoci</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCustomPlot" name="wilgoc_wykres" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(170, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>811</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>51</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 53, 95);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>1011</width>
             <height>33</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_powrot">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#pushButton_powrot{
	border -radius: 3px;
	background: rgb(109, 109, 109);
	color: rgb(255, 255, 255)
}

QPushButton#pushButton_powrot:hover{
	background-color: rgb(47, 47, 47);
	color: rgb(255, 255, 255)
}

QPushButton#pushButton_powrot:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255)
}
</string>
              </property>
              <property name="text">
               <string>Powrót</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="nazwa_okna00">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel#nazwa_okna00{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
	border: 1px solid blue;
    border-radius: 2px;
}
font: 9pt &quot;Segoe Print&quot;;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Wykresy pomiarowe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>31</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
